bot_starter_pack/src/config_tests.rs

```diff
@@ -0,0 +1,23 @@
+#[allow(unused_imports)]
+use crate::bot_utils::{TestsConfig, TestsTarget};
+
+// Look at the example below
+
+pub fn tests_configuration() -> TestsConfig {
+    vec![
+        // Define your targets here
+    ]
+    .into()
+}
+
+/*  Example
+
+pub fn tests_configuration() -> TestsConfig {
+    vec![
+        TestsTarget::new("../TEST-CONTRACTS/out/xyz.sol/xyz.json")
```

Comment:
This is the "testing toolkit" - You specify a list of test files and list out all the detectors that you want to test against them. (here it is Example and AnotherExample detectors)

When `cargo test` is ran, the detectors will receive their respective list of test files (so relationship inversion is taken care of)